spec/users/user_manages_cart_spec.rb

```diff
@@ -35,22 +27,23 @@
     # factory girl? stubbing current user? wrap the cart in a method called cart instead of directly accessing the cart and that would allow me to stub the method the same way we stubbed current user before.
   end
 
-  # it "allows users to remove things from their cart" do
-  # As an Unauthenticated User
-  # When I visit /cart
-  # And I click “Remove from Cart” on an Item
-  # And I see a flash alert that “The item has been removed from your cart”
-  # And I see the item disappear from my Cart
-  # Then I should see the cart size decrease by one item
-  # end
-
-  # it "allows users to view their cart" do
-  # end
+  it "allows users to remove things from their cart" do
+    create(:item)
+    visit "/menu"
+    click_link_or_button "Add to Cart"
+    visit "/cart"
+    click_link_or_button "Remove from Cart"
+    expect(page).to have_content("The item has been removed from your cart")
+    expect(page).to_not have_content("salted caramel")
+  end
 
-  # it "allows users to increase the quantity of an item in their cart" do
-  # As an Unauthenticated User
-  # When I visit /cart
-  # And I increase the quantity of an item in my cart
-  # Then the size of my cart should increase by one
-  # end
+  it "allows users to increase the quantity of an item in their cart" do
+    create(:item)
+    visit "/menu"
+    click_link_or_button "Add to Cart"
+    visit "/cart"
+    click_link_or_button "Add Another"
+    expect(page).to have_content("Quantity: 2")
+    expect(page).to have_content("Total Items: 2")
+  end
 end
```

Comment:
As Josh pointed out, all the tests in this file start with the exact same lines of code. We need to find a way to abstract that away.